CPC-CHARTER.md

```diff
@@ -233,51 +233,33 @@ election is required if there is only one candidate and no objections to
 the candidates election. Elections shall be done within the Projects by
 the Collaborators active in the Project.
 
+The CPC will elect from amongst CPC members:
```

Comment:
Just voting, or is this inclusive of regular?

Reply:
Good catch. My intent was that this should be inclusive of regular members. I clarified it.